k8s/service_controller/model/service_instance.go

```diff
@@ -22,9 +22,9 @@ type LastOperation struct {
 }
 
 type CreateServiceInstanceRequest struct {
```

Comment:
Sure, I just put them backwards in there depending on how to organize. I think:
sc/model/...
sb/model/...
makes sense, not wedded to either one. Just that we should use go packages for these rather than baking in the name of the objects (sc_ & sb_)